components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -394,6 +394,13 @@ Rules of type `Node` can have conditions with attributes `Chef Organization`, `C
 Navigate to the project list page once more. Select `Update Projects`. Upon completion of the update, you should be able to
 filter by `project-devops` across Automate's dashboards and see only the ingested data that you expect.
 
+#### Effortless Infra Project
+To create a project that contains all Effortless Infra nodes, add a node ingest rule with a condition that uses an attribute of Chef Server and a value of "localhost".
+
+![](/images/docs/effortless-project-rule.png)
+
+The above rule matches on a node's Chef Server field, which is set to "localhost". This works because all Effortless Infra nodes do not have a Chef Server. Then, by adding a second condition that matches a specific Chef Policy name, subgroups (i.e., nodes matching a specified policy name) of Effortless Infra nodes can be created. 
```

Comment:
@lancewf A question about "This works because all Effortless Infra nodes do not have a Chef Server."

 In this case, is 'localhost' considered the default value of a `Chef Server` attribute? And thus, it is not set to a different value because a `Chef Server` attribute isn't used in Effortless Infra?

Reply:
@mjingle You could think of "localhost" as the default value, but it is closer to when Effortless Infra is used that is the value that Chef Server is set to. This is because Effortless Infra does not have a Chef Server. When Effortless Infra is not used, the IP address or domain name of the Chef Server is used in place of the "localhost".